mmv1/third_party/terraform/tests/resource_firestore_database_update_test.go.erb

```diff
@@ -0,0 +1,89 @@
+<% autogen_exception -%>
+package google
+
+<% unless version == 'ga' -%>
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccFirestoreDatabase_update(t *testing.T) {
+	t.Parallel()
+
+	orgId := getTestOrgFromEnv(t)
+	randomSuffix := randString(t, 10)
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"random": {},
```

Comment:
I might not understand what is happening here, but do you know if you need this line? I didn't see the `random` provider being used by your test.

Reply:
Not needed after all, thanks!